### PR TITLE
Replace hand-drawn commutative diagrams with TeX

### PR DESCRIPTION
**Pull request by mseri**
I had some spare time and decided to give it a try. This should solve https://github.com/hmemcpy/milewski-ctfp-pdf/issues/75

The diagrams are written in tikzcd, that should be the state of the art for commutative diagrams as far as I know.

It does rewrite only the diagrams that did not contain colours, these are in chapters 3.2, 3.9, 3.12.
There are some on the last chapter but are more complex and from my first attempts they look ugly. I will submit a separate PR for those once I am satisfied with the result.

### CODE REVIEW

**Comment by hmemcpy** [comment]
I love it! Thank you for the effort!
I'll leave this PR open for a few days until (hopefully) @BartoszMilewski and others can verify it...

Thanks again, looks fantastic!

image

**Comment by mseri** [comment]
Thanks! No problem. Let me know if there are things that need fixing.

I have added another commit to improve the pagination of the diagrams at page 424. The `id.jpg` image on chapter 28.3 was much bigger (in terms of relative size of the "font") than the previous one, making it slightly smaller and comparable with the other moves the diagrams there on the same page instead of leaving a huge white gap on the page.

**Comment by hmemcpy** [comment]
That's great! Yeah, I've been tweaking the image sizes myself to have them fit on the page...

**Comment by mseri** [comment]
I've found the time to fix also the ones in chapter 30. This should cover all of them.

File: src/content/3.12/Enriched Categories.tex [hunk 1]
```diff
@@ -165,20 +183,48 @@ \section{Enriched Category}
...
+        & \cat{C}(a,b)
+    \end{tikzcd}
+  \end{subfigure}
+  \hspace{1cm}
+  \begin{subfigure}
+    \centering
+    \begin{tikzcd}[row sep=large]
+      i \otimes \cat{C}(a,b)
+      \arrow[rr, "j_b \otimes \id"]
+      \arrow[dr, "\rho"]
```
Comment (hmemcpy) [inline src/content/3.12/Enriched Categories.tex]: I think this should be a `\lambda`:
image

**Comment by hmemcpy** [comment]
I think I found just one "issue" with the diagrams (should be a lambda instead). If you change that, I'll happily mere! Also, feel free to add your name to the list of contributors in the README :)

Thanks again, and happy New Year!

**Comment by mseri** [comment]
Done! Thank you very much, and happy new year to you as well!

**Comment by hmemcpy** [comment]
Amazing! Going to release a new pdf right now :)

**Pull request merged**